app/models/event.rb

```diff
@@ -41,6 +41,7 @@ class Event < ActiveRecord::Base
   scope :canceled, -> { where(state: 'canceled') }
   scope :withdrawn, -> { where(state: 'withdrawn') }
   scope :highlighted, -> { where(is_highlight: true) }
+  scope :current, -> { joins(:event_type).where('start_time <= ?', Time.current).map { |e| e if e.end_time > Time.current }.compact }
```

Comment:
Scheduled talks that get cancelled after they have been scheduled, still keep the start_time set, so that we can inform our attendees accordingly (that this talk has been cancelled) (introduced in #1082)

If the talk has been cancelled, and there is no replacement, we don't want to show it here.
So here you should only be taking into consideration proposals in state **confirmed**.